src/box/vy_lsm.c

```diff
@@ -888,6 +888,17 @@ vy_lsm_rotate_mem(struct vy_lsm *lsm)
 	return 0;
 }
 
+int
+vy_lsm_check_rotate_mem(struct vy_lsm *lsm)
```

Comment:
I'd call it sort of `vy_lsm_rotate_on_demand` or `vy_lsm_rotate_if_required`. Feel free to skip

Reply:
Renamed to `vy_lsm_rotate_if_required`.